capreolus/registry.py

```diff
@@ -105,7 +105,7 @@ def _create_ingredient(cls, module, sub_ingredients, command_list):
 
         # create module config consisting of (1) the module class name and (2) its config options (from config())
         ingredient.add_config({"_name": cls.name})
-        ingredient.config(cls.config)
+        ingredient.config(cls.config) # should be ingredient.config(cls.cfg)?
```

Comment:
No, experiment.config and ingredient.config are used to specify the module's config function (which defines config options and their default values). `cls.cfg` will be populated by the Pipeline with the module's active config.